be/src/services/entity-service.ts

```diff
@@ -3,16 +3,33 @@ import {instance} from "./persistence.service";
 import {Entity} from "../generated";
 import {Collection} from "mongodb";
 
-export default class EntityService {
+class EntityService {
     private readonly persistenceService = instance;
     private readonly entityCollection: string = "entities";
 
     private get collection(): Collection {
         return this.persistenceService.db.collection(this.entityCollection);
     }
 
-    async createEntity(vocabID: string, entity: Entity): Promise<any> {
-        return Promise.resolve(null);
+    private createNewDBO(vocabID: string, entity: Entity): Entity {
+        return {
+            id: null,
+            created: new Date().toISOString(),
```

Comment:
```suggestion
            created: new Date()
```
The date object will be converted by mongoDB itself, here no conversion is needed

Reply:
Das wird so nichts, da new Date() eine Zahl erzeugt und in dem created ein ISO String eingetragen werden muss

Reply:
It is possible. `EntityDto` object has `created: DateTime`, which is of type `string`,  but in the Dbo version it is `created: Date`, which allows the syntax: `created: new Date()` by creating a new instance of entity.